lib/class-wp-block.php

```diff
@@ -107,8 +107,8 @@ public function __construct( $block, $available_context = array(), $registry = n
 
 		$this->available_context = $available_context;
 
-		if ( ! empty( $this->block_type->context ) ) {
```

Comment:
does anyone know is this is used anywhere else? I unfortunately am not familiar with block context code.

Reply:
It should be rather `usesContext`, it doesn't error when this context is not provided.

We would need to update all occurrences of `context` in `block.json` files of core blocks.

Reply:
Why? Isn't this just an internal switch which parses `context` from block.json, turns it into `usesContext`, and then the `WPBlock` class takes `usesContext` to set the `context` property on itself?

Reply:
i.e. doesn't the external API would stay the same?